components/admin/admin-stats-overview.tsx

```
@@ -1,21 +1,39 @@
+const STATS_REGION_LABEL = "Key performance metrics";
+const LOADING_LABEL = "Loading statistics data";
 
 interface AdminStatsOverviewProps {
   stats: AdminStats | undefined;
   isLoading: boolean;
 }
 
 export function AdminStatsOverview({ stats, isLoading }: AdminStatsOverviewProps) {
+  if (isLoading) {
+    return (
+      <div role="region" aria-label={LOADING_LABEL}>
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<section>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)